openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -11,7 +10,7 @@
 from celery.utils.log import get_task_logger
 from celery_utils.logged_task import LoggedTask
 from django.conf import settings
-from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
```

Comment:
Since I was in here making changes I fixed this to bring it in line with Django's best practices (to not import the `user` model directly).